native/acter/api.rsh

```diff
@@ -2160,6 +2160,14 @@ object PublicSearchResultItem {
     fn join_rule_str() -> string;
     // fn room_type() -> Option<RoomType>;
     fn room_type_str() -> string;
+
+    /// whether to have avatar
+    fn has_avatar() -> bool;
+
+    /// get the binary data of avatar
+    /// if thumb size is given, avatar thumbnail is returned
```

Comment:
There's been on and off discussion around the avatar thumbnails about their source of generation and how matrix treats thumbnail objects. Does matrix generates media thumbnail on its own on the backend or we have to explicitly upload its version from client ??